src/components/Breadcrumb/style.styl

```diff
@@ -1,31 +1,33 @@
 @require 'settings/breakpoints'
 
 .Breadcrumb
-    margin-bottom 1rem
-    &::after
-        content ''
-        clear left
-        display block
-
-    &__title
-        color coolGrey
-
-    &__crumb
-        float left
-
-    &__last &__title
-        color black
-
-    &__separator
-        padding-left .3em
-        padding-right .3em
-
-    &__link
-        cursor pointer
-
-    +small-screen()
-        width 100% // for ellipsis to work correctly
-        &__crumb&__last
-            width 100% // for ellipsis to work correctly
-        &__crumb:not(&__last)
-              display none
+    display flex
+    align-items center
+
+
+.Breadcrumb__previousButton
+    margin-right 1.5rem
+    cursor pointer
+
+.Breadcrumb__items
+    flex-grow 1
+    display flex
+    flex-direction column
```

Comment:
Why the need for flex here ?

Reply:
That's totally useless actually. I will remove it. Thanks